src/main/java/com/nike/cerberus/operation/core/UpdateStackOperation.java

```diff
@@ -75,6 +74,9 @@ public void run(final UpdateStackCommand command) {
         } else if (Stack.DATABASE.equals(command.getStack())) {
             // TODO: implement storing password if it was changed
             //configStore.storeCmsDatabasePassword(databasePassword);
+        } else if (Stack.LOAD_BALANCER.equals(command.getStack())) {
```

Comment:
We may be able to refactor these if/else statements later on.

They don't seem all that bad now, but looks like they'll continue to grow larger and more involved at which point it might be better to have separate commands or some other organization.

Reply:
Agreed, we should break up update-stack into multiple commands, one per stack makes sense to me.

Reply:
(Though not part of this PR)

Reply:
Sounds like a good future improvment